### PR TITLE
replace therubyracer→mini_racer

### PR DESCRIPTION
**Pull request by T-Y89**
ref https://github.com/rubyjs/therubyracer?tab=readme-ov-file#status-unmaintained

and I specify rails version at 7.0.0 to prevent regression

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,6 @@
 ![logo](docs/assets/Bucky-Management.png)
 [![Maintainability](https://api.codeclimate.com/v1/badges/aaeb28e229926007442b/maintainability)](https://codeclimate.com/github/lifull-dev/bucky-management/maintainability)
 
-## Overview
```
Comment (nk-ty) [inline README.md:5]: この部分を削除した理由を教えて下さい！

Could you tell me why this part was removed?

Comment (T-Y89) [reply README.md]: I accidentally deleted this file, but I restored it

**Review by nk-ty — APPROVED** [review]

**Comment by T-Y89** [comment]
@nk-ty こんにちは

**Comment by T-Y89** [comment]
@shouan2212 
こんにちは

**Pull request merged**